lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -1498,19 +1561,21 @@ def test_lifecycle(
         cm = CacheManager(server_config, make_logger)
         archive = cm.archive_root / "ABC"
         cache = cm.cache_root / md5
+        dataset_name = source_tarball.name[:-7]
```

Comment:
I realize that you only moved this line, but don't we prefer `Dataset.stem()` over the slice idiom?  (This is one of only two places remaining in the code....)